Makefile

```diff
@@ -334,9 +334,6 @@ $(DEVICE_FARM_UPLOAD_IPA): $(XCTESTRUN_PACKAGE) | $(DEVICE_TEST_PATH) $(PAYLOAD_
 	# Creating IPA package for upload
 	cp -R $(BUILT_DEVICE_PRODUCTS_DIR)/$(APP_NAME).app $(PAYLOAD_DIR)
 
-	# Test moving frameworks for AWS to re-codesign (if they exist)
-	mv $(PAYLOAD_DIR)/$(APP_NAME).app/Frameworks/*.framework/Frameworks/*.framework $(PAYLOAD_DIR)/$(APP_NAME).app/Frameworks || true
```

Comment:
Noting for history that this has been replaced by including MapboxCoreMaps & MapboxCommon at the top level within the test host app (in the Xcode project)